tox.ini

```diff
@@ -12,12 +12,12 @@ commands = flake8 SCAutolib/ test/ setup.py
 deps =
     -rrequirements.txt
     pytest
+    pytest-env
 commands = pytest test/ -sv -m "not ipa and not service_restart" {posargs}
```

Comment:
slightly unrelated comment: I find this scary and fragile since the tests are destructive.
https://jwodder.github.io/kbits/posts/pytest-mark-off suggests ways to skip them by default and not rely no not forgetting to specify a filter.

Reply:
hm, point here that these tests would be executed only in CI and actually do no destructive. Also, tests are executed in a fresh container which is removed after execution. And if talking about execution on the local machine, the user just should not do this and test everything in a dedicated environment. we can discuss this in more details